llvm/test/MC/StubV/add.s

```diff
@@ -8,13 +8,13 @@
 # CHECK-INST-NEXT:  <MCOperand Reg:13>>
 add a0, a1, a2
 # CHECK-ENCODING: encoding: [0x67,0x80,0x00,0x00]
-# CHECK-INST:      <MCInst #296 JALR
+# CHECK-INST:      <MCInst #{{[0-9]+}} JALR
```

Comment:
why is there a check for JALR about the ret?

Reply:
It's because RET is an alias in assembly printing/parsing, and isn't represented as its own MCInst